### PR TITLE
Update panel to support SRE-3b HV diagnostics

### PR DESCRIPTION
**Pull request by RustyPedrosa**
Add HVIL Term Sense indicator.  
Add inverter override: force disable (for calibration) and force enable (skips RTD process - not working).  
Add DI voltage/current charts.  
Add TPS/BPS voltages, TPS % chart.  
Add temp files to gitignore.

### CODE REVIEW

**Comment by RustyPedrosa** [comment]
Please assign reviewers @TimothyLe

File: SRE2.dsk [hunk 1]
```diff
@@ -88,7 +88,7 @@ ObjectKind={FB44BF1E-7C70-4F04-A5B5-A476C4006598}
 Visible=1
 Left=18
 Top=1
-Width=1795
+Width=1823
```
Comment (TimothyLe) [inline SRE2.dsk:91]: Why is the width bigger?

Comment (TimothyLe) [reply SRE2.dsk:91]: @Chi-Ho rEVIEW THIS

**Review by TimothyLe — APPROVED** [review]
DI - Drive Inverter
Charts are a PCAN Explorer file

**Pull request merged**